### PR TITLE
Merge Development

### PR DESCRIPTION
**Pull request by ImBreadd**
Description

### CODE REVIEW

File: CMP1903M Assessment 1 Base Code/Program.cs [hunk 1]
```diff
@@ -29,7 +29,7 @@ static void Main()
 
 
             //TO ADD: Get the frequency of individual letters?
-            Console.WriteLine("Hello");
+
```
Comment (valrossenOliver) [inline CMP1903M Assessment 1 Base Code/Program.cs:32]: Example of comment on specific change

**Review by valrossenOliver — APPROVED** [review]
Example of code review

File: CMP1903M Assessment 1 Base Code/Program.cs [hunk 2]
```diff
@@ -29,7 +29,7 @@ static void Main()
 
 
             //TO ADD: Get the frequency of individual letters?
-            Console.WriteLine("Hello");
```
Comment (ImBreadd) [inline CMP1903M Assessment 1 Base Code/Program.cs:32]: sfdsffsfr

**Review by ImBreadd — COMMENTED** [review]
hhttrtyhrt

**Review by valrossenOliver — CHANGES_REQUESTED** [review]
Add description on pull request

**Comment by ImBreadd** [comment]
> 

oof

**Comment by ImBreadd** [comment]
> Add description on pull request

oof

**Pull request merged**